nacc/uds3/fvp/builder.py

```diff
@@ -12,8 +12,140 @@
 def build_uds3_fvp_form(record):
     """ Converts REDCap CSV data into a packet (list of FVP Form objects) """
     packet = fvp_packet.Packet()
-    
-    #Set up the forms.
+
+    # Set up the forms
+    add_z1_or_z1x(record, packet)
+    add_a1(record, packet)
+    if record['fu_a2_sub'] == '1' or record['fu_a2sub'] == '1':
+        add_a2(record, packet)
+    if record['fu_a3_sub'] == '1' or record['fu_a3sub'] == '1':
+        add_a3(record, packet)
+    if record['fu_a4_sub'] == '1' or record['fu_a4sub'] == '1':
+        add_a4(record, packet)
+    if record['fu_b1_sub'] == '1' or record['fu_b1sub'] == '1':
+        add_b1(record, packet)
+    add_b4(record, packet)
+    if record['fu_b5_sub'] == '1' or record['fu_b5sub'] == '1':
+        add_b5(record, packet)
+    if record['fu_b6_sub'] == '1' or record['fu_b6sub'] == '1':
+        add_b6(record, packet)
+    if record['fu_b7_sub'] == '1' or record['fu_b7sub'] == '1':
+        add_b7(record, packet)
+    add_b8(record, packet)
+    add_b9(record, packet)
+    add_c1s_or_c2(record, packet)
+    add_d1(record, packet)
+    add_d2(record, packet)
+    clsform.add_cls(record, packet, fvp_forms)
+    update_header(record, packet)
+
+    return packet
+
+
+def add_z1_or_z1x(record, packet):
+    # Forms A1, A5, B4, B8, B9, C2, D1, and D2 are all REQUIRED.
```

Comment:
it doesn't look like form A5 is required for fvps. it's not in this builder file, nor is it in nacc's fvp ded. this comment also doesn't specify that C1S is required for ptids with Z1 forms, and C2 is required for ptids with Z1X forms.
i was just confused about forms that seemed to be missing, during testing, is all, so the comment (and the fact that it's the exact same comment as in the ivp builder, when there is the difference in A5 forms) felt misleading.